native_client/deepspeech_compat.h

```diff
@@ -0,0 +1,97 @@
+#ifndef DEEPSPEECH_COMPAT_H
+#define DEEPSPEECH_COMPAT_H
+
+#include "deepspeech.h"
+
+/**
+ * @brief An object providing an interface to a trained DeepSpeech model.
+ *
+ * @param aModelPath The path to the frozen model graph.
+ * @param aNCep UNUSED, DEPRECATED.
+ * @param aNContext UNUSED, DEPRECATED.
+ * @param aAlphabetConfigPath The path to the configuration file specifying
+ *                            the alphabet used by the network. See alphabet.h.
+ * @param aBeamWidth The beam width used by the decoder. A larger beam
+ *                   width generates better results at the cost of decoding
+ *                   time.
+ * @param[out] retval a ModelState pointer
+ *
+ * @return Zero on success, non-zero on failure.
+ */
+int DS_CreateModel(const char* aModelPath,
+                   unsigned int /*aNCep*/,
+                   unsigned int /*aNContext*/,
+                   const char* aAlphabetConfigPath,
+                   unsigned int aBeamWidth,
+                   ModelState** retval)
+{
```

Comment:
Maybe we should issue some build-time warning about using deprecated interface ?

Reply:
Like `#warning "YOU ARE RELYING ON DEPRECATED INTERFACE, PLEASE UPDATE YOUR CODE."`